payloads/payloads_darwin_amd64.go

```diff
@@ -0,0 +1,8 @@
+package payloads
+
+import "embed"
+
+// Darwin payloads separated by architecture to avoid duplicate payloads when cross compiling
+
+//go:embed build/darwin/amd64/*/*
```

Comment:
Should we put the `build` directory in this package? vs the root (i.e. `build/`) or subdirectory from where the source is `llm/build`?

Reply:
OK, I'll give this a shot.